app/views/admin/people/show.html.erb

```diff
@@ -25,7 +25,7 @@
 
     <div class="gem-c-summary-list--with-image app-view-people-details__summary_list">
       <%= render "govuk_publishing_components/components/summary_list", {
-        title: "Details",
+        title: tag.span("Details", class: "gem-c-heading govuk-heading-l govuk-!-margin-bottom-0"),
```

Comment:
we can pass heading size and level here

https://components.publishing.service.gov.uk/component-guide/summary_list/with_custom_section_heading